### PR TITLE
Fix README links and centering.

### PR DESCRIPTION
**Pull request by kern**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<center><img src="https://redspread.com/images/logo.svg" alt="logo" width= "400"/>
+<p align="center"><img src="https://redspread.com/images/logo.svg" alt="logo" width="400" /></p>
```
Comment (zfogg) [inline README.md]: idk I would argue that this isn't semantically paraphrase text at all

**Comment by mfburnett** [comment]
@kern can you update to master so I can merge this?

File: README.md (hunk 1 above)
Comment (kern) [inline README.md]: GitHub strips `<div />`s bruh

Comment (kern) [inline README.md]: And while we're nitpicking, it's a "paragraph," not a "paraphrase": https://www.w3.org/TR/html-markup/p.html

**Comment by kern** [comment]
rebased and force-pushed because I like living on the edge and keeping your commits clean :3

File: README.md (hunk 1 above)
Comment (mfburnett) [inline README.md]: Hah you two...appreciate the attention to detail.

**Comment by zfogg** [comment]
"paraphrase" was a typo lol c'mon you know I'm all about semantic web

**Comment by kern** [comment]
then be semantically correct ;)

**Comment by zfogg** [comment]
😭

**Pull request merged**